app/src/main/java/com/strv/movies/network/ApiModule.kt

```diff
@@ -0,0 +1,23 @@
+package com.strv.movies.network
+
+import com.strv.movies.network.auth.AuthApi
+import dagger.Module
+import dagger.Provides
+import dagger.hilt.InstallIn
+import dagger.hilt.components.SingletonComponent
+import retrofit2.Retrofit
+import retrofit2.create
+import javax.inject.Singleton
+
+@Module
+@InstallIn(SingletonComponent::class)
+object ApiModule {
+
+    @Provides
+    @Singleton
+    fun provideMovieApi(@UnauthorizedRetrofit retrofit: Retrofit): MovieApi = retrofit.create()
+
+    @Provides
+    @Singleton
+    fun provideAuthApi(@UnauthorizedRetrofit retrofit: Retrofit): AuthApi = retrofit.create()
```

Comment:
Shouldn't this one be **@AuthorizedRetrofit**?

Reply:
Nope. You are not authorized when signing in :D